__tests__/app.test.js

```diff
@@ -94,6 +94,93 @@ describe('Testing app', () => {
                 })
             })
         });
+        test('allows the data to be sorted by any column, but the default is still date', () => {
+            const sortBy = 'author';
+            return request(app)
+            .get(`/api/articles?sort_by=${sortBy}`)

```

Comment:
Ideally i'd check that the other columns can be sorted by. 

It would be better to discover the bug/they dont sort now, than when you're on front end and cant work out why you sort by isn't letting you sort by certain columns.

Reply:
I've added another test which is sortBy = 'title'. That works and to double check I misspelt title too to make sure it fails.